src/vs/workbench/contrib/webview/browser/webviewEditor.ts

```diff
@@ -136,15 +132,6 @@ export class WebviewEditor extends BaseEditor {
 
 	public focus(): void {
 		super.focus();
-		if (!this._onFocusWindowHandler) {
```

Comment:
This code was added to fix a bug: when you switch back to vscode (using cmd+tab or some other action without clicking) we need to refocus on the webview if it were previously active. Removing it will cause that to break

@sbatten may have some ideas on how this code could be tweaked so that clicking on menus bars (or other controls) works as expected

Reply:
@mjbvz I don't repro the issue of not focusing the webview with @ZainChen's branch after alt-tabbing back it. Can you still repro the issue with his changes @mjbvz? If we do need this code still, what has focus in the alt-tab case? Can we check that before trying to steal focus. If it is a consistent element, we can try to limit the impact here.

Reply:
I am thinking about a question. Is it OK to remove this part of the code (0.0)?
Because: remove the code, if you click on the blank area above the vscode window switch back to to vscode, the focus is on the webview. If you click on the webview window switch back to vscode, the focus is also on the webview. Click on another area to return to vscode, the focus is not on the webview.

Reply:
Hello, if my description is the same as the actual situation, it seems that don't have to click all the areas to go back to vscode and put the focus on the webview. What do you think of this idea?@mjbvz @sbatten
![image](https://user-images.githubusercontent.com/15615524/56708055-f447ce00-674d-11e9-8b46-3b052efc97a8.png)